src/assets/AssetRegistrar.sol

```diff
@@ -44,8 +44,12 @@ contract AssetRegistrar is DBC, Owned, AssetRegistrarInterface {
     function getSymbol(address ofAsset) constant returns (string) { return information[ofAsset].symbol; }
     function getDecimals(address ofAsset) constant returns (uint) { return information[ofAsset].decimal; }
 
-    /// @notice Get human-readable information about an Asset
+    /// @notice Gets human-readable information about an Asset
     /// @param ofAsset address for which descriptive information is requested
+    /// @return Human-readable name of the Asset as in ERC223 token standard
+    /// @return Human-readable symbol of the Asset as in ERC223 token standard
+    /// @return Url for extended information of the asset
+    /// @return Same as url but for ipfs
```

Comment:
would do this in doxity return https://github.com/DigixGlobal/doxity#quickstart format